src/Controllers/HomeController.cs

```diff
@@ -1,21 +1,35 @@
 ﻿using System.Diagnostics;
+using System.Collections.Generic;
 using GovUk.Education.SearchAndCompare.UI.ViewModels;
 using GovUk.Education.SearchAndCompare.UI.Services;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using GovUk.Education.SearchAndCompare.UI.ActionFilters;
+using Microsoft.Extensions.Configuration;
 
 namespace GovUk.Education.SearchAndCompare.UI.Controllers
 {
     //[Authorize]
     public class HomeController : CommonAttributesControllerBase
     {
+        private bool PreLaunchMode;
+
+        public HomeController(IConfiguration Configuration)
+        {
+            PreLaunchMode = !string.IsNullOrWhiteSpace(Configuration["SITE_PASSWORD"]);
+        }
+
         [HttpGet("")]
         [HttpGet("home")]
         [HttpGet("home/index")]
         public IActionResult Index()
         {
-            return View("Index");
+            if (PreLaunchMode) {
+                return View("Index");
+            } else {
+                bool hideBack = true;
```

Comment:
I would of just set hideback in viewbag here, then on view check if its set if it is set check if its true to hide

Reply:
Didn't know you could do that, ugh!

Reply:
Unfortunately this doesn't seem to work. The state of the `ViewBag` doesn't get persisted across controllers...